### ISSUE TITLE
HTTP fetch error message

### ISSUE THREAD

**Issue by saemiw** [opengisch/QField#778]
On Android 5 tablets running QField 1.3.4 I regularly get the following error message "HTTP fetch..."
v:/QField is the file path where our "mother project" is saved. The project file that is triggering this error on QField was however set offline with QFieldSync. I don't see why it should want to access this directory (but I also have no understanding what http fetch is/does).

![Screenshot_2020-01-07-14-17-46](https://user-images.githubusercontent.com/20398621/71904520-91c12180-3166-11ea-9561-4197e7f6f3d7.png)

**Comment by signedav** [comment]
Only on Android 5 @saemiw ?
Can you provide a demo project (or does it occur with the one we already have to reproduce the former crash?)

**Comment by saemiw** [comment]
> 
> 
> Only on Android 5 @saemiw ?
> Can you provide a demo project (or does it occur with the one we already have to reproduce the former crash?)

Yes only Android 5 and yes you should already have such a project. Let me know if you need one again...

**Comment by koenigherford** [comment]
You can reproduce the issue in this way:
1)
1a) Create a new template-project in QGIS with a sample-layer and configure attribute form as "drag and drop designer". 
1b) ->Export it to local disk
1c) ->copy it to mobile device
1d) ->open it there 
1e) -> **no issue** 
1f) ->**CLOSE** qfield

2)
2a) Open template-project from before (1a)
2b) ->change attribute-form to "provide ui-file" 
2c) ->select a ui-file **on a server** (NOT local file like "c:") (im my case "K:/GIS/QGIS-Projekte/König/temp/test_44_form.ui" (while K: is representing the server //nt230dbg) as  "provide ui-file"
![grafik](https://user-images.githubusercontent.com/28433436/173832243-ba565997-22de-4e98-845e-d66a340d8e1d.png)
2d) ->change attribute form as "drag and drop designer" (path setting from 2c) will be disabled)
2e) ->Export project to local disk
2f) ->copy project to mobile device
2g) ->open it there 
2h) -> **warning message**
![grafik](https://user-images.githubusercontent.com/28433436/173833301-fc143115-3a05-4d82-98c4-5f0ad5352c95.png)
2i) ->**CLOSE** qfield

3)
3a) Take project from 2a
3b) ->change attribute-form to "provide ui-file" 
3c) ->manually remove path to server from widget
3d) ->change attribute form as "drag and drop designer" (path setting from 2c) will be disabled)
3e) ->Export project to local disk
3f) ->copy project to mobile device
3g) ->open it there 
3h) -> **no issue** 

I think QGIS stores unused links to ui-file. In case of a **link to a server** this will cause the errors within QFIELD - altrough path to ui-file is not used at all (because its a drag and drop attribute form-setting).

Usig Samsung Galaxy Tab Active 3, Android 12, QFieldSync 4.1.1., QField 1.9.6, QGIS 3.22.2
